### ISSUE TITLE
No module named ansiblelint.__main__; 'ansiblelint' is a package and cannot be directly executed

### ISSUE THREAD

**Issue by ssbarnea** [willthames/ansible-lint#148]
```
No module named ansiblelint.__main__; 'ansiblelint' is a package and cannot be directly executed
```

Python modules that act as utility scripts are supposed to also be loaded using the module approach like ```
ansible-lint --version

python2 -m ansiblelint --version

```
```

**Comment by ssbarnea** [comment]
These are fixed by PR#149

**Comment by willthames** [comment]
Fixed in v3.4.8

**Comment by ssbarnea** [comment]
Thank you!
